src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    private final Random random = new Random();
+
+    public Lottery(int number, String color) {
+        super(number, color);
+    }
+
+    @Override
+    public String toString() {
+        return super.toString();
+    }
+
+    public Ball getRandomBall() {
+        int number = random.nextInt(100);
+        return new Ball(number,"" + getRandomColor());
```

Comment:
Don't add an empty String. Better [use name() for getting String representation of enum constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants). Fix in all places